webapp/certification/views.py

```diff
@@ -12,4 +12,69 @@
 
 
 def certification_home():
-    return flask.render_template("certification/index.html")
+
+    certified_releases = api.certified_releases(limit="0")["objects"]
+    certified_makes = api.certified_makes(limit="0")["objects"]
+
+    # Desktop section
+    desktop_releases = []
+    desktop_vendors = []
+
+    for release in certified_releases:
+        if int(release["desktops"]) > 0 or int(release["laptops"]) > 0:
+            desktop_releases.append(release)
+
+    for vendor in certified_makes:
+        if int(vendor["desktops"]) > 0 or int(vendor["laptops"]) > 0:
+            desktop_vendors.append(vendor)
+
+    # Server section
```

Comment:
If all you need for the overview are the counts you can refactor the for loop to be
```
    # Server section
    server_releases = {}
    server_vendors = api.vendor_summaries_server()["vendors"]

    for vendor in server_vendors:
        for release in vendor["releases"]:
            if release in server_releases:
                server_releases[release] += vendor[release]
            else:
                server_releases[release] = vendor[release]
```

Reply:
It would also be interesting to know if we could just use the output of `https://certification.canonical.com/api/v1/certifiedreleases/?servers__gte=1`. The counts seem slightly different between the two endpoints so it might be that we need to use `vendor_summaries`

Reply:
Not sure what you mean but I did the math and it matches exactly